templates/macros/helpers.html

```diff
@@ -12,6 +12,12 @@
   {{attr|xmlattr}}
 {%- endmacro -%}
 
+{%- macro format_paragraphs(paragraphs) -%}
+  {%- for paragraph in paragraphs -%}
+    <p>{{paragraph}}</p>
+  {%- endfor -%}
+{%- endmacro -%}
+
```

Comment:
```suggestion
{%- macro format_paragraphs(paragraphs) -%}
  {%- for paragraph in paragraphs -%}
    <p>{{paragraph}}</p>
  {%- else -%}
  {%- endfor -%}
{%- endmacro -%}
```

This macro should deal with empty iterable objects